pepper-apis/src/test/java/org/broadinstitute/ddp/route/ParticipantsLookupRouteTest.java

```diff
@@ -0,0 +1,125 @@
+package org.broadinstitute.ddp.route;
+
+import static io.restassured.RestAssured.given;
+import static java.util.Collections.emptyList;
+import static org.hamcrest.Matchers.equalTo;
+
+import java.util.ArrayList;
+
+import io.restassured.http.ContentType;
+import io.restassured.mapper.ObjectMapperType;
+import org.broadinstitute.ddp.TxnAwareBaseTest;
+import org.broadinstitute.ddp.constants.RouteConstants;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.ddp.db.dao.AuthDao;
+import org.broadinstitute.ddp.db.dao.JdbiClient;
+import org.broadinstitute.ddp.filter.StudyAdminAuthFilter;
+import org.broadinstitute.ddp.filter.TokenConverterFilter;
+import org.broadinstitute.ddp.json.admin.participantslookup.ParticipantsLookupPayload;
+import org.broadinstitute.ddp.json.admin.participantslookup.ParticipantsLookupResultRow;
+import org.broadinstitute.ddp.security.JWTConverter;
+import org.broadinstitute.ddp.service.participantslookup.ParticipantsLookupResult;
+import org.broadinstitute.ddp.service.participantslookup.ParticipantsLookupService;
+import org.broadinstitute.ddp.transformers.NullableJsonTransformer;
+import org.broadinstitute.ddp.util.TestDataSetupUtil;
+import org.hamcrest.Matchers;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import spark.Spark;
+
+public class ParticipantsLookupRouteTest extends TxnAwareBaseTest {
+
+    private static String urlTemplate;
+    private static TestDataSetupUtil.GeneratedTestData testData;
+
+    @BeforeClass
+    public static void setupServer() {
+        var jsonSerializer = new NullableJsonTransformer();
+
+        int port = RouteTestUtil.findOpenPortOrDefault(5559);
+        Spark.port(port);
```

Comment:
This is great!! I think we should do more of these unit tests that spin up their own Spark server so things are more isolated.